src/transformer/section-category.ts

```diff
@@ -328,8 +330,11 @@ export const chooseSectionCategory = (
 ): SectionCategory | undefined => {
   const secType = section.getAttribute('sec-type') as SecType
   const secCat = chooseSectionCategoryByType(secType)
+  const parent = section.parentNode?.parentNode?.nodeName.toLowerCase()
   if (secCat) {
     return secCat
+  } else if (parent !== 'body') {
+    return 'MPSectionCategory:subsection'
   } else {
```

Comment:
can we have this logic in either the `jats-dom-parser.ts` when handling the section tag, or even before?